doc/user/sql-grammar/sql-grammar.bnf

```diff
@@ -174,6 +174,14 @@ format_spec ::=
   'CSV WITH' ('HEADER' ( '(' col_name (',' col_name)* ')' ) | n 'COLUMNS') ('DELIMITED BY' char)? |
   'TEXT' |
   'BYTES'
+key_strat ::=
+  'KEY STRATEGY' strat
+val_strat ::=
+  'VALUE STRATEGY' strat
+strat ::=
+  'INLINE' avro_reader_schema |
+  'ID' schema_registry_id |
```

Comment:
are `avro_reader_schema` and `schema_registry_id` tied to types at all?

Reply:
no, those don't lead anywhere. But that's consistent with how the BNF works elsewhere; I don't think we have a good way to express that one has to be a string and one has to be an int.